workspaces/mcp-integrations/plugins/mcp-actions-backend-backend/src/routers/createSseRouter.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2025 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import PromiseRouter from 'express-promise-router';
+import { Router } from 'express';
+import { McpService } from '../services/McpService';
+import { SSEServerTransport } from '@modelcontextprotocol/sdk/server/sse.js';
+import { HttpAuthService } from '@backstage/backend-plugin-api';
+
+/**
+ * Legacy SSE endpoint for older clients, hopefully will not be needed for much longer.
+ */
```

Comment:
a happy side effect of having to provide this copy is that if we need to revisit the SSE connections issues of the past we can at least add debug in here to assist the debug

Reply:
Yup, in general this should be a pretty useful playground for us as we experiment with potential upstream contributions (e.g. the prompting you mentioned on slack today)